src/app.jsx

```diff
@@ -205,6 +205,16 @@ class Application extends React.Component {
                                 return { containersStats: copyStats };
                             });
                         }
+                    } else {
+                        // HACK: Sometimes we don't get 'remove' event when container is removed
+                        // https://github.com/containers/libpod/issues/6664
```

Comment:
This bug got fixed 3 weeks ago, so it's worth doing another test run without this hack.